source/MaterialXCore/Value.cpp

```diff
@@ -254,6 +254,60 @@ ScopedFloatFormatting::~ScopedFloatFormatting()
     Value::setFloatPrecision(_precision);
 }
 
+StringVec parseStructValueString(const string& str)
+{
+    const char SEPARATOR = ';';
```

Comment:
Should we leave these constant character definitions inside the function, or move them up to the anonymous namespace above, where they would become part of the static initialization of `Value.cpp`, and could be assigned more unique names?  I don't have a strong opinion on this, and wanted to mention it as an option for consideration.

Reply:
I made declared them `static` so they'll get initialized the first time the function gets called - but it feels easier to read if we keep them defined close to where they are used.